locales/en.default.schema.json

```diff
@@ -1978,6 +1978,162 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "slideshow": {
+      "name": "Slideshow",
+      "settings": {
+        "layout": {
+          "label": "Layout",
+          "options__1": {
+            "label": "Full width"
+          },
+          "options__2": {
+            "label": "Grid"
+          },
+          "options__3": {
+            "label": "Grid and peek"
+          }
+        },
+        "slide_height": {
+          "label": "Slide height",
+          "options__1": {
+            "label": "Adapt to first image"
+          },
+          "options__2": {
+            "label": "Small"
+          },
+          "options__3": {
+            "label": "Medium"
+          },
+          "options__4": {
+            "label": "Large"
+          }
+        },
+        "slider_visual": {
+          "label": "Slider visual",
+          "options__1": {
+            "label": "Numbers"
+          },
+          "options__2": {
+            "label": "Dots"
+          },
+          "options__3": {
+            "label": "Counter"
+          }
+        },
+        "heading_size": {
+          "label": "Heading font size",
+          "options__1": {
+            "label": "Medium"
+          },
+          "options__2": {
+            "label": "Large"
+          }
+        },
+        "text_align": {
+          "label": "Text alignment",
+          "options__1": {
+            "label": "Top left"
+          },
+          "options__2": {
+            "label": "Top center"
+          },
+          "options__3": {
+            "label": "Top right"
+          },
+          "options__4": {
+            "label": "Middle left"
+          },
+          "options__5": {
+            "label": "Middle center"
+          },
+          "options__6": {
+            "label": "Middle right"
+          },
+          "options__7": {
+            "label": "Bottom left"
+          },
+          "options__8": {
+            "label": "Bottom center"
+          },
+          "options__9": {
+            "label": "Bottom right"
+          }
+        },
+        "options": {
+          "content": "Options"
+        },
+        "show_text_box": {
+          "label": "Show text box"
+        },
+        "image_overlay_opacity": {
+          "label": "Image overlay opacity"
+        },
+        "auto_rotate": {
+          "label": "Auto-rotate slides"
+        },
+        "change_slides_speed": {
+          "label": "Change slides every"
+        },
+        "mobile": {
+          "content": "Mobile"
+        },
+        "show_text_below": {
+          "label": "Show text below images"
+        },
+        "accessibility": {
+          "content": "Accessibility",
+          "label": "Slideshow description"
```

Comment:
I am reflecting on this one, it might need some additional helper text for merchants.

Reply:
Agreed. For example, video content we have the following hint text:

> "Describe the video to make it accessible for customers using screen readers."

Perhaps we could include something like,

> "A brief description of the slideshow for customers using screen readers."